gemrb/plugins/OpenALAudio/OpenALAudio.h

```diff
@@ -90,6 +90,27 @@ struct AudioStream {
 struct CacheEntry {
 	ALuint Buffer;
 	tick_t Length;
+
+	CacheEntry(ALuint buffer, tick_t length) : Buffer(buffer), Length(length) {}
+	CacheEntry(const CacheEntry&) = delete;
+	CacheEntry& operator=(const CacheEntry&) = delete;
```

Comment:
Reminder to add proper move methods in order to prevent `Buffer` from hanging around more than once (also SDLAudio) in case of some implicit movement.